dune-project

```diff
@@ -22,4 +22,6 @@
   containers
   fmt
   (cmdliner (>= 1.1.0))
+  diffutils
   (ocamlformat (and :with-dev-setup (= 0.26.2)))))
+
```

Comment:
This looks like a rebase error, these changes shouldn't be here.

Reply:
> This looks like a rebase error, these changes shouldn't be here.

I noticed that this also appears in the api-watch.opam file. I think it might need similar attention.